providers/universal-provider/src/providers/eip155.ts

```diff
@@ -45,6 +46,8 @@ class Eip155Provider implements IProvider {
         return parseInt(this.getDefaultChain()) as unknown as T;
       case "wallet_getCapabilities":
         return (await this.getCapabilities(args)) as unknown as T;
+      case "wallet_getCallsStatus":
+        return (await this.getCallStatus(args)) as unknown as T;
```

Comment:
Wouldn't this result in a unhandled promise rejection if the call throws ?

Reply:
no, it as http calls are try/catched. Only if the request can't be sent to thte wallet and/or is rejected by the wallet it would throw